File: 2012/November/19-11-2012/fizzbuzz.rb
1.upto(100) do |i|
  if i % 5 == 0 and i % 3 == 0
    p "FizzBuzz"
  elsif i % 5 == 0
    p "Buzz"
  elsif i % 3 == 0
    p "Fizz"
  else
    p i
  end
end

File: 2012/November/26-11-2012/src/FizzBuzz.js
function FizzBuzz(){
  this.calculate = function(number){
    if(dividablebyThree(number) && dividablebyFive(number)){
      return 'fizzbuzz';
    }
    if(dividablebyThree(number)){
      return 'fizz';
    }
    if(dividablebyFive(number)){
      return 'buzz';
    }

    return number.toString();
  }

  function dividablebyThree(number){
    return number % 3 == 0 && number != 0;
  }

  function dividablebyFive(number){
    return number % 5 == 0 && number != 0;
  }
}